src/script/hooks/useDropFiles.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useEffect} from 'react';
+
+import {isDragEvent} from '../guards/Event';
+
+const onDragOver = (event: Event) => event.preventDefault();
+
+const useDropFiles = (selector: string, onDropOrPastedFile: (files: File[]) => void) => {
+  const onDropFiles = (event: Event) => {
+    if (isDragEvent(event)) {
+      event.preventDefault();
```

Comment:
do we only want to prevent default if the event is the right type or in general? right now, if the event is different, default is executed